File: 2020/day01a.py
def main():
    lines = open('input.txt', 'r').read().splitlines()
    for i in lines:
        for e in lines:
            if int(i) + int(e) == 2020:
                print(int(i) * int(e))
                quit()

if __name__ == '__main__':
    main()

File: 2022/day01a.jl
function main()
    lines = readlines(open("input.txt"))

    max_cal = 0
    sum_cal = 0

    for line in lines
        if line != ""
            sum_cal += parse(Int, line)
        else
            if sum_cal > max_cal
                max_cal = sum_cal
            end

            sum_cal = 0
        end
    end

    print("$max_cal\n")
end

main()
